code/game/objects/items/weapons/implants/implant_prescan.dm

```diff
@@ -0,0 +1,48 @@
+/obj/item/implant/grey_autocloner
+	name = "technocracy cloning bio-chip"
+	desc = "Allows for advanced instantanious cloning!"
+	icon_state = "generic"
+	origin_tech = "materials=3;combat=5;syndicate=2"
+	activated = FALSE
+	trigger_causes = BIOCHIP_TRIGGER_DEATH_ANY
+	var/obj/machinery/grey_autocloner/linked
+	var/datum/dna2/record/our_record
+
+/obj/item/implant/grey_autocloner/death_trigger(mob/source, gibbed)
+	imp_in.ghostize(TRUE)
+	if(linked)
+		linked.growclone(our_record)
+
+/obj/item/implant/grey_autocloner/implant(mob/source, mob/user, force)
+	if(!linked)
+		atom_say("Please link the implanter with a Technocracy cloning pod!")
+		return 0
+	. = ..()
+	if(.)
+		if(ishuman(imp_in))
```

Comment:
You could probably guard clause this on `if(!. || !ishuman(imp_in) return`

Reply:
Not allowing implantation with this

Reply:
I might be stupid, ignore me

Reply:
done